src/App.tsx

```diff
@@ -1,204 +1,103 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
+import { Status } from './types/Status';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 96;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(Status.all);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [errorMessage, setErrorMessage] = useState<Errors | null>(null);
+
+  const onAllClick = useCallback(() => {
+    if (filterBy !== Status.all) {
+      setFilterBy(Status.all);
+    }
+  }, [filterBy]);
+
+  const onActiveClick = useCallback(() => {
+    if (filterBy !== Status.active) {
+      setFilterBy(Status.active);
+    }
+  }, [filterBy]);
+
+  const onCompletedClick = useCallback(() => {
+    if (filterBy !== Status.completed) {
+      setFilterBy(Status.completed);
+    }
+  }, [filterBy]);
```

Comment:
you can make code shorter. Just for example: 
```ts
const onStatusClick = useCallback((status) => {
  if (filterBy !== status) {
    setFilterBy(status);
  }
}, [filterBy]);
```

Reply:
agree) please update to Nazarii comment